### ISSUE TITLE
/var/log/message is flooded with Keepalived_healthcheckers messages when misc_dynamic is used.

### ISSUE THREAD

**Issue by anta5010** [acassen/keepalived#677]
Hi,

  Our MISC_CHECK tests use misc_dynamic, and measure the response times, so they are always
wobbling slightly.  Keepalived_healthcheckers produces several messages per second in /var/log/messages:

```
Oct 18 17:17:47 us1 Keepalived_healthcheckers[13080]: pid 29053 exited with status 253
Oct 18 17:17:47 us1 Keepalived_healthcheckers[13080]: pid 29055 exited with status 254
Oct 18 17:17:47 us1 Keepalived_healthcheckers[13080]: pid 29057 exited with status 255
Oct 18 17:17:47 us1 Keepalived_healthcheckers[13080]: pid 29073 exited with status 255
Oct 18 17:17:47 us1 Keepalived_healthcheckers[13080]: pid 29075 exited with status 255
Oct 18 17:17:47 us1 Keepalived_healthcheckers[13080]: pid 29106 exited with status 253

```
  
  Can these messages be changed to LOG_DEBUG when misc_dynamic is used?

**Comment by pqarmitage** [comment]
Which version of keepalived are you using? Commit c736fa03, which was first included in v1.3.6, should have resolved this problem.

**Comment by anta5010** [comment]
Oh, thanks. I'll check. At the moment I'm using 1.3.5 from Centos7.4
